api/src/main/java/io/strimzi/api/kafka/model/KafkaAuthorizationOpa.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.DescriptionFile;
+import io.strimzi.crdgenerator.annotations.Example;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+/**
```

Comment:
Maybe change this comment to mention OPA, what it is and a link to the docs?

Reply:
I improved the comment. TBH I'm not sure about the doc link - we do not have them anywhere else and it sounds like something what will stop working after some time without being noticed.